settings.gradle

```
@@ -15,7 +15,7 @@ dependencyResolutionManagement {
     }
 }
 
-rootProject.name = 'micronaut-graal-languages'
+rootProject.name = 'micronaut-graal-languages-parent'
```

why do you change this? 

```suggestion
rootProject.name = 'micronaut-graal-languages'
```